clap_complete/src/dynamic/completer.rs

```diff
@@ -311,6 +311,23 @@ fn complete_arg_value(
     let mut values = Vec::new();
     debug!("complete_arg_value: arg={arg:?}, value={value:?}");
 
+    let (prefix, value) = match arg.get_value_delimiter() {
+        Some(delimiter) => match value {
+            Ok(value_os) => match value_os.rfind(delimiter) {
+                Some(pos) => {
+                    // SAFETY: `pos + delimiter.len_utf8()` is guaranteed to be within the bounds of `value_os`
+                    // `pos` is the first byte index of the last match of delimiter
+                    // `delimiter.len_utf8()` is the length of the delimiter in bytes
+                    let (prefix, value) = value_os.split_at(pos + delimiter.len_utf8());
```

Comment:
Where is the `split_at` coming from for `OsStr`?  Would `clap_lex::OsStrExt::split` (taking the last value) or adding a `rsplit_once` work for your needs?

Reply:
Sorry, it's a mistaken variable naming. `value_os` is a `&str`